test/unit/lbp-wrapper.js

```diff
@@ -55,78 +58,48 @@ describe("Contract: LBPWrapper", async () => {
 
       sortedTokens = sortTokens(setup.tokenList);
       tokenAddresses = sortedTokens.map((token) => token.address);
+      JOIN_KIND_INIT = 0;
```

Comment:
Looks like you're reassigning a constant here, when in fact it is not really a constant.